src/content/app/tools/blast/components/listed-blast-submission/ListedBlastSubmission.tsx

```diff
@@ -39,6 +40,10 @@ export type Props = {
   submission: BlastSubmission;
 };
 
+export const FailedSubmissionHelpText = () => (
+  <p>Unable to get results for this sequence</p>
+);
```

Comment:
When a submission is collapsed, the text doesn't really make sense:

![image](https://user-images.githubusercontent.com/6834224/208383831-b55094ba-2f88-419e-9342-2c33ee39cf17.png)

We may choose to:
- keep this text in the collapsed view regardless (and wait for Andrea to come back and correct it)
- replace it with a more appropriate text that we write ourselves
- remove the question button in the collapsed view (until Andrea comes back and corrects it)

Reply:
Question - under what circumstances can, in one submission, one or more sequences fail and one or more succeed. I can see how we might get no alignments for the one or more sequences, but I don't know what could cause one or more to fail

Reply:
I don't have a rational answer. Those hypothetical errors are probably the most improbable, especially if it's for several sequence-species combinations, like in [this XD](https://xd.adobe.com/view/47c05579-d7aa-414a-a0ca-e2b5e2c88636-1293/screen/566d3729-59d6-444a-8774-66d84af759dc?fullscreen) (1 sequence against 2 species). But then, shouldn't we proceed from the assumption that any job, for whatever reason, however rarely, might fail?

Reply:
We are discussing how to deal with something that we have no reason to expect to happen. Could / should we have a generic way to deal with these hypothetical situations (I agree we need something) but focus more on the situations that we know will fail ?

Reply:
These are questions for Andrea. So far we've got designs for the following scenarios:

1) Something happened to a submission as a whole ([link](https://xd.adobe.com/view/47c05579-d7aa-414a-a0ca-e2b5e2c88636-1293/screen/0070e1b1-554a-4f96-afd0-4d620e592019?fullscreen))

![image](https://user-images.githubusercontent.com/6834224/208700152-7deccaa7-c249-4f12-9bc5-622705c8f711.png)

2) Something happened to one or more jobs grouped by a sequence, and represented visually as a box ([link](https://xd.adobe.com/view/47c05579-d7aa-414a-a0ca-e2b5e2c88636-1293/screen/ca48bfae-0a80-4d9d-b8d9-133f3c8db809?fullscreen), although see [another link](https://xd.adobe.com/view/47c05579-d7aa-414a-a0ca-e2b5e2c88636-1293/screen/3d44f722-8823-417f-ba94-ff02b9144f0e?fullscreen) as a variation):

![image](https://user-images.githubusercontent.com/6834224/208700899-5edb3572-f384-4843-b202-dfaf232ec544.png)

3) Something happened to one or more jobs inside of an expanded box ([link](https://xd.adobe.com/view/47c05579-d7aa-414a-a0ca-e2b5e2c88636-1293/screen/88c148e3-c947-4203-b1ca-fcd52901c4a4?fullscreen)):

![image](https://user-images.githubusercontent.com/6834224/208701107-83497cc6-987d-4597-b110-11a55473c872.png)

Looks pretty generic to me.